chaos_genius/views/anomaly_data_view.py

```diff
@@ -62,12 +63,21 @@ def kpi_anomaly_detection(kpi_id):
                 }
             )
 
+        dimensions_values = _get_dimensions_values(kpi_id)
+
         period = kpi_info["anomaly_params"]["anomaly_period"]
         hourly = kpi_info["anomaly_params"]["frequency"] == "H"
 
         end_date = get_anomaly_output_end_date(kpi_info)
 
-        anom_data = get_overall_data(kpi_id, end_date, period)
+        dimension = request.args.get("dimension", default=None)
+        value = request.args.get("value", default=None)
+
+        if dimension and value:
```

Comment:
Do you need to check if `dimension` is `Overall` (and `value` is `None`) too?

Reply:
No. Like we decided, for overall, both dimension and value will be set to None. FE won't return anything.

Reply:
Got it